src/Microsoft.AspNetCore.Mvc.Razor/Properties/AssemblyInfo.cs

```diff
@@ -3,5 +3,7 @@
 
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Microsoft.AspNetCore.Mvc.RazorPages, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
Should leave out these additional IVT changes until and unless we change pubternal classes in Mvc.Razor.

Reply:
Need this in a follow up PR - https://github.com/aspnet/Mvc/pull/8071

Reply:
Then do this in #8071

Reply:
Leaving as is.